packages/apps/crm/src/components/organisms/DropdownContactView/DropdownContactView.tsx

```diff
@@ -28,6 +28,9 @@ interface DropdownContactViewProps {
   mobilePhone?: string;
   emailAddress: string;
   webSite: string;
+  name?: string;
+  lastName?: string;
+  company?: string;
```

Comment:
Maybe this can be wrap under networkData (or researchData, or something else) to make check of data easier. 
Please add check in component if return condition